prow/plugins.yaml

```diff
@@ -132,7 +132,10 @@ blockades:
   - kubernetes/community
   blockregexps:
   - ^events/2016
+  - ^events/2017
+  - ^events/2018
```

Comment:
```suggestion
```
We can't put this in yet, as 2018 isn't over (contrib summit)

Reply:
"Applied suggestion" and squashed.